web/assets/js/components/gameoptions.ts

```diff
@@ -279,13 +329,26 @@ const Gameoptions = class extends Component {
       ease: 'steps(9)',
       duration: 0.4
     });
+
+    const dir: number = e ? (e.currentTarget.dataset.ctrl === '+' ? 1 : -1) : 0;
+    this.timer = Math.min(
+      this.timers[this.options.category].length - 1,
+      Math.max(0, this.timer + dir)
+    );
+    this._updateTimer('', 0);
+    gsap.to('.gameoptions-sprite', {
+      backgroundPosition: cat ? '100%' : '0',
+      ease: 'steps(9)',
+      duration: 0.4
+    });
```

Comment:
@alexiscolin 
Just double checking, is this out of place?

Reply:
I see it's an addition to the PR, but I don't remember ever adding it

Reply:
You are right it's a duplicate. I'm gonna remove it. I've already fixed some others.